nbsite/cmd.py

```diff
@@ -8,6 +8,15 @@
 
 from .util import copy_files
 
+DEFAULT_PYVIZ_ORDERING = [
```

Comment:
Shouldn't that be DEFAULT_SITE_ORDERING?  It doesn't seem specific to PyViz, and nbsite doesn't seem like it should be mentioning pyviz specifically.

Reply:
Sure.